components/speedreader/rust/lib/Cargo.toml

```diff
@@ -31,7 +31,7 @@ flate2 = { version = "1.0.19", features = ["rust_backend"], default-features = f
 # lifeguard <= 0.6.0 panics at runtime on Rust 1.48+
 lifeguard = "^ 0.6.1"
 # url must be pinned to the same version as adblock
```

Comment:
This one isn't going to work as per the comment here, at least until https://github.com/brave/brave-browser/issues/11364 is sorted out